utils/markdown/inlines.go

```diff
@@ -549,7 +549,7 @@ func (p *inlineParser) parseAutolink(c rune) bool {
 		RawDestination: linkRange,
 		markdown:       p.markdown,
 	})
-	p.position += (link.End - link.Position)
+	p.position += link.End - link.Position
```

Comment:
If this is not changed by "gofmt -s -w", I am not sure if this is worth the code churn.

Reply:
no, that was suggested by Goland and there is no additional value by the brackets